### PR TITLE
feat(profile): support password change/reset for einfra namespace

### PR DESCRIPTION
**Pull request by zlamalp**
- Support password change and reset operations on a profile page
  for login-namespace: einfra.
- Password reset redirects to pwd reset application.
- Password change displays custom dialog with appropriate password
  checks and requires old password.

### CODE REVIEW

File: perun-wui-profile/src/main/resources/cz/metacentrum/perun/wui/profile/client/resources/PerunProfileTranslation_cs.properties [hunk 1]
```diff
@@ -27,6 +27,10 @@ addCert=Přidat nový certifikát
 
 loadingUserData=Načítám data o uživateli
 updateEmail=aktualizovat
+changePassword=změnit heslo
+resetPassword=resetovat heslo
```
Comment (HejdaJakub) [inline perun-wui-profile/src/main/resources/cz/metacentrum/perun/wui/profile/client/resources/PerunProfileTranslation_cs.properties:31]: Just about consistency - in the English version, you use the first upper case letter. The same thing at updateEmail.
```suggestion
changePassword=Změnit heslo
resetPassword=Resetovat heslo
```

**Review by HejdaJakub — COMMENTED** [review]
Just minor text suggestions. Otherwise, it looks good.

File: perun-wui-core/src/main/java/cz/metacentrum/perun/wui/client/resources/PerunTranslation.java [hunk 2]
```diff
@@ -291,6 +291,38 @@ public interface PerunTranslation extends Messages {
 	@DefaultMessage("Password <b>can`t contain login, name or surname</b>, not even backwards!")
 	public String einfraPasswordStrengthForNameLogin();
 
+	@DefaultMessage("New password can`t be empty!")
+	public String passwordCantBeEmpty();
+
+	@DefaultMessage("Passwords doesn`t match!")
```
[the remark this answers is not in the document]
Comment (zlamalp) [reply perun-wui-core/src/main/java/cz/metacentrum/perun/wui/client/resources/PerunTranslation.java]: Fixed.

File: perun-wui-profile/src/main/resources/cz/metacentrum/perun/wui/profile/client/resources/PerunProfileTranslation_cs.properties (hunk 1 above)
Comment (zlamalp) [reply perun-wui-profile/src/main/resources/cz/metacentrum/perun/wui/profile/client/resources/PerunProfileTranslation_cs.properties]: Fixed also with translation for update email button.

**Pull request merged**